### PR TITLE
remove the code that supports case insensitivity. Fix tests that have…

### PR DESCRIPTION
**Pull request by sjwells**
… slice append issue

### CODE REVIEW

File: mapper/uri_utils_test.go [hunk 1]
```diff
@@ -1,55 +1,63 @@
...
-var locationURIs = append(conceptURIs, baseURI+"Location")
+func allLabelsFor(label string) []string {
+	allLabels := []string{}
+	for t := label; t != ""; t = ParentType(t) {
+		allLabels = append(allLabels, t)
+	}
+	return allLabels
+}
+
+var thingLabels = allLabelsFor("Thing")
```
Comment (Zapadlo) [inline mapper/uri_utils_test.go]: Would it be cleaner to:
```go
var (
    thingLabels = allLabelsFor("Thing")
    conceptLabels = allLabelsFor("Concept")
    ...
)
```
?

Comment (sjwells) [inline mapper/uri_utils_test.go]: Sounds sensible yes

**Comment by Zapadlo** [comment]
+1

**Comment by scott-ace-newton** [comment]
:+1:

**Pull request merged**